<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewEditUsers</class>
 <widget class="QDialog" name="ViewEditUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </widget>
  <widget class="QListWidget" name="userList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="infoTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>File</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correct</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Missed</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date &amp; Time</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fileFilter">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>File: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ViewEditUsers</receiver>
   <slot>onBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ViewEditUsers</receiver>
   <slot>on_userChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ViewEditUsers</receiver>
   <slot>onDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileFilter</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ViewEditUsers</receiver>
   <slot>on_fileChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBack()</slot>
  <slot>on_userChanged(QString)</slot>
  <slot>onDelete()</slot>
  <slot>on_fileChange(QString)</slot>
 </slots>
</ui>
